src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+
+const collectionOfButton = [...thead.rows[0].cells];
+
+for (let i = 0; i < collectionOfButton.length; i++) {
+  collectionOfButton[i].addEventListener('click', () => {
+    [...tbody.rows]
+      .sort((rowA, rowB) => {
+        const salaryA = rowA.lastElementChild.textContent.replace(/[$,]/g, '');
+        const SalaryB = rowB.lastElementChild.textContent.replace(/[$,]/g, '');
```

Comment:
Ensure that the salary values are correctly parsed into numbers before performing the subtraction for sorting. Currently, the `replace` method removes the dollar sign and commas, but the result is still a string. Consider using `parseFloat` or `Number` to convert the cleaned string into a number.